frontend/src/wizards/Placement/PlacementSection.tsx

```diff
@@ -253,21 +262,24 @@ export function PlacementSection(props: {
       )}
       {placementCount === 0 && placementRuleCount === 0 && placementBindingCount === 1 && (
         <WizItemSelector selectKey="kind" selectValue={PlacementBindingKind}>
-          <Select
-            path="placementRef.name"
-            label={t('Placement')}
-            required
-            hidden={(binding) => binding.placementRef?.kind !== PlacementKind}
-            options={namespacedPlacements.map((placement) => placement.metadata?.name ?? '')}
-          />
-          <WizSingleSelect
-            path="placementRef.name"
-            label={t('Placement rule')}
-            placeholder={t('Select the placement rule')}
-            required
-            hidden={(binding) => binding.placementRef?.kind !== PlacementRuleKind}
-            options={namespacedPlacementRules.map((placement) => placement.metadata?.name ?? '')}
-          />
+          {usesPlacementRule ? (
+            <WizSingleSelect
+              path="placementRef.name"
+              label={t('Placement rule')}
+              placeholder={t('Select the placement rule')}
+              required
+              hidden={(binding) => binding.placementRef?.kind !== PlacementRuleKind}
```

Comment:
I think you can remove the `hidden` props since now you are conditionally rendering instead.

Reply:
Good point. Thanks!